README.md

```diff
@@ -81,11 +81,8 @@ Requirements
     with the version 3.5 of the framework, but CSML was initially
     developped against version 2.0.
 
-  - Alternatively, CSML should work under Linux with Mono and
-    hopefully the newest .NET Core framework (again probably with
-    minor adjustments to the build system).  Support for Mono was only
-    tested in the early days in the project and never used in
-    production, though.
+  - Alternatively, CSML has been under Linux with Mono 4.x.
```

Comment:
AFAIU (and I'm really an expert of this ecosystem), I think these are two different things: Mono implements the .NET Framework, and .NET Core is a new framework (which happens to be very close from .NET) re-designed to be cross-platform.  So I'd rephrase as:

> Alternatively, CSML has been tested under Linux with Mono 4.x, but not used in production.
>
> Moreover, it is expected that CSML could be made to work with little effort with the newest .NET Core framework both under Windows and other systems.